packages/shared/src/hooks/useProfileMenu.ts

```diff
@@ -0,0 +1,18 @@
+import { useContextMenu } from 'react-contexify';
+
+export default function useProfileMenu(): {
+  onMenuClick: (e: React.MouseEvent) => void;
+} {
+  const { show } = useContextMenu({ id: 'profile-context' });
+
+  const onMenuClick = (e: React.MouseEvent) => {
+    const { right, bottom } = e.currentTarget.getBoundingClientRect();
+    show(e, {
+      position: { x: right - 160, y: bottom + 4 },
```

Comment:
Try to see if you can use here a calculated value or at least a constant value that is shared between the files instead of just setting it to `160`